user/build-environment-updates/2019-01-10.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Build Environment Update History
+layout: en
+permalink: /user/build-environment-updates/2019-01-10
+category: linux_build_env_updates
+date: 2019-01-10
+---
+
+# 2019-01-10
+
+## Changes
+- Language: generic is recognized
+- System Info is more readable
+- PhantomJS2 is available
+- Tmate is preinstalled
+- Gem (and rvm) are configured to use `--no-document` rather than `--no-ri` and `--no-rdoc`.
+- Ensure ip forwarding on localhost does not get interrupted once per day
```

Comment:
would it be good for context to link to the public issue / forum post about this?

Reply:
I'm +1 on that